pkg/webui/console/containers/device-onboarding-form/index.js

```diff
@@ -33,50 +44,159 @@ import DeviceTypeFormSection, {
 } from './device-type-form-section'
 import validationSchema from './validation-schema'
 
-const initialValues = merge({}, provisioningInitialValues, typeInitialValues)
+const initialValues = merge(
+  {
+    _registration: REGISTRATION_TYPES.SINGLE,
+  },
+  provisioningInitialValues,
+  typeInitialValues,
+)
 
 const DeviceOnboardingFormInner = () => {
   const {
-    values: {
-      frequency_plan_id,
-      lorawan_version,
-      lorawan_phy_version,
-      ids: { join_eui },
-    },
+    values: { frequency_plan_id, lorawan_version, lorawan_phy_version, ids },
   } = useFormContext()
+
   const maySubmit =
     Boolean(frequency_plan_id) &&
     Boolean(lorawan_version) &&
     Boolean(lorawan_phy_version) &&
-    join_eui.length === 16
+    ids?.join_eui?.length === 16
```

Comment:
```suggestion
    typeof _claim === 'boolean'
```
Otherwise the submit button appears already before confirming the EUI.